### PR TITLE
[DC-733] Fixup file browser tests for React 18

### PR DESCRIPTION
**Pull request by nawatts**
This resolves warnings that appear in a few file tests with upgrades to React 18 and React Testing Library v14.

### CODE REVIEW

File: src/pages/AzurePreview.test.js [hunk 1]
```diff
@@ -183,7 +183,7 @@ describe('AzurePreview', () => {
 
           // Act
           const submitButton = screen.getByText('Submit');
-          await act(() => user.click(submitButton));
+          await user.click(submitButton);
```
Comment (nawatts) [inline src/pages/AzurePreview.test.js:186]: Not related to the file browser but didn't want to make a separate PR.

**Comment by cahrens** [comment]
What is the warning you get in this case? It's hard for me to imagine that the unneeded wrapping with act actually hurts anything, so I'm curious what issue it causes.

**Review by cahrens — APPROVED** [review]

**Comment by cahrens** [comment]
> What is the warning you get in this case? It's hard for me to imagine that the unneeded wrapping with act actually hurts anything, so I'm curious what issue it causes.

Never mind, saw the explanation in another PR.

_The newer versions appear to be unhappy with userEvent methods wrapped in act. That causes warnings about the testing environment not being configured for act._

**Review by s-rubenstein — APPROVED** [review]
LGTM

**Pull request merged**